addons/kubernetes-dashboard.md

```diff
@@ -1,13 +1,16 @@
 # Kubernetes Dashboard
 
-The NAIS platform comes with the default [Kubernetes dashboard](https://github.com/kubernetes/dashboard)
-running in its own namespace in each cluster. It is also running without an ingress, so you will need to use
-[port-forwarding](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/)
-to interact with it.
+The NAIS platform comes with the default [Kubernetes dashboard][1] running in its own namespace in each cluster.
+It is also running without an ingress, so you will need to use [port-forwarding][2] to interact with it.
 
-With [kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/) 1.10 and later, the following command can be run:
+With [kubectl][3] 1.10 and later, the following command can be run:
```

Comment:
we can probably remove the version disclaimer now

Reply:
> Wow, very nice job!
> 
> One comment about extracting the links as references. I think it's this is most useful in large documents with heavy re-use of URLs, but for most of our pages we usually use a link once. When updating or reading the doc in as .md, we now have to do a extra hop to see which link being used - which might be unnecessary.
> 
> What's the rationale behind this?

The link references were used purely to cut down on line length. In some cases I believe it leads to a more readable document, and in other cases it is pretty useless and causes more work as an editor to find out where a link leads to. 

I'll admit the line length thing is a personal crusade of mine, and perhaps a hard limit is not that cool in the end; Gitbook doesn't replace newlines in lists with spaces (although that should be the case in most Markdown renderers..). I'm also unsure whether link references should be recommended/not recommended, or used only where it's sensible. The latter will be difficult to maintain, since it will be up to the editor.